ops/deploy-ccs.groovy

```diff
@@ -125,13 +125,15 @@ def buildAppAmis(String gitBranchName, String gitCommitId, AmiIds amiIds, AppBui
 			writeFile file: "${workspace}/ops/ansible/playbooks-ccs/extra_vars.json", text: """{
     "data_server_launcher": "${workspace}/${appBuildResults.dataServerLauncher}",
     "data_server_war": "${workspace}/${appBuildResults.dataServerWar}",
-    "data_pipeline_zip": "${workspace}/${appBuildResults.dataPipelineZip}"
+    "data_pipeline_zip": "${workspace}/${appBuildResults.dataPipelineZip}",
+    "git_branch": "${gitBranchName}"
```

Comment:
Make sure to pass along `gitBranchName` here during build time for the clone repo role.